kafka-ui-react-app/src/components/Schemas/Details/__test__/__snapshots__/SchemaVersion.spec.tsx.snap

```diff
@@ -1,35 +1,246 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SchemaVersion matches snapshot 1`] = `
-<Fragment>
-  <tr>
-    <td
-      style={
-        Object {
-          "width": "3%",
-        }
+<ContextProvider
+  value={
+    Object {
+      "alert": Object {
+        "color": Object {
+          "error": "#FAD1D1",
+          "info": "#E3E6E8",
+          "success": "#D6F5E0",
+          "warning": "#FFEECC",
+        },
+        "shadow": "rgba(0, 0, 0, 0.1)",
+      },
+      "buttonStyles": Object {
+        "fontSize": Object {
+          "L": "16px",
+          "M": "14px",
+          "S": "14px",
```

Comment:
is it correct that M and S should be the same size?

Reply:
It is a snapshot @NelyDavtyan